### PR TITLE
Add i18n support

### PR DESCRIPTION
**Pull request by dannycolin**
This patch adds support for internationalization.

### CODE REVIEW

File: src/background_script.js [hunk 1]
```diff
@@ -78,12 +78,16 @@ function updateBadgeStatus(currentTab) {
     browser.browserAction.setIcon({
       path: "assets/badge-indicator-on.svg",
     });
-    browser.browserAction.setTitle({ title: "Chrome Mask is on." });
+    browser.browserAction.setTitle({
+      title: browser.i18n.getMessage("statusOn")
```
Comment (denschub) [inline src/background_script.js:82]: The string here is called `maskStatusOn`. Same with the off state below.

**Review by denschub — CHANGES_REQUESTED** [review]
Good work, thanks for working on this!

The title update is broken (see inline-comment), and `npm run lint` please.

**Review by denschub — APPROVED** [review]

**Comment by denschub** [comment]
Thanks! I won't release this now, but I'll try to get this translated into German soon, and maybe I'll push a release then!

**Comment by dannycolin** [comment]
> Thanks! I won't release this now, but I'll try to get this translated into German soon, and maybe I'll push a release then!

Sounds good. I'll try to translate it in French before end of week ;)

**Pull request merged**